list.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*elems)
+    @list = elems
```

Comment:
- [ ] Great job working on this milestone. When I run ruby main.rb from the terminal, I get error and when I run ruby list.rb i get no response. I am unable to start your application. Kindly review and if possible provide instructions in your README how a user can start the application.
![Screenshot from 2023-07-28 09-05-42](https://github.com/gisachris/Enumerables/assets/62849872/f2e9f6ba-75dc-4916-8a14-f6d479862f60)

Reply:
@t-yanick Thank you for the great insight and review! we will solve the bug and send it for another review.